src/styles/base/_utils.scss

```diff
@@ -150,8 +150,3 @@
 .link-color {
   color: var(--link-color);
 }
-
-// Error color
-.error-text {
```

Comment:
Have we removed it because it is available in mattermost?

Reply:
No, because it is already present in this file above.

Reply:
Got it, thanks!